Bugfix 15865 Fixed description of is not among

bhall2001 opened the pull request:
fixed description to correctly state that is not among implies string
does not exist.

bhall2001 committed:
Merge remote-tracking branch 'livecode/develop' into develop

bhall2001 committed:
[[ Bug 15865 ]] Fixed is not among description

fixed description to correctly state that is not among implies string
does not exist.

livecodeali commented on docs/notes/bugfix-15865:
```diff
@@ -0,0 +1 @@
+# Fixed Dictionary example for is within
```
@bhall2001 I think you've used the same bugfix note description as for the other PR!

Also, could you make sure the file has a `.md` suffix please?

bhall2001 committed:
Added .md and corrected Notes

Added in .md on the notes file and changed the notes text to correctly
describe the fix ;-)

livecodeali commented:
@livecode-vulcan review ok fb329e6

livecode-vulcan commented:
💙 review by @livecodeali ok fb329e6a4a23913cd64b73b4e8f1f5fcca243549

livecode-vulcan commented:
😎 test success fb329e6a4a23913cd64b73b4e8f1f5fcca243549
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/376)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/381)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/376)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/376)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/382)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/376)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/376)
